src/geometry/CTiglBSplineAlgorithms.cpp

```diff
@@ -852,7 +853,11 @@ math_Matrix CTiglBSplineAlgorithms::bsplineBasisMat(int degree, const TColStd_Ar
     bspl_basis.Init(0.);
     for (Standard_Integer iparm = 1; iparm <= params.Length(); ++iparm) {
         Standard_Integer basis_start_index = 0;
+#if OCC_VERSION_HEX >= VERSION_HEX_CODE(7,1,0)
+        BSplCLib::EvalBsplineBasis(derivOrder, degree + 1, knots, params.Value(iparm), basis_start_index, bspl_basis);
```

Comment:
Did you run the unit tests of tigl to check if this is still correct? This function is tested for correct values.

Reply:
no i did not run the tests, i let the CI services do that^^

Reply:
Good thought but does not help here, since the unit tests don't run with OpenCASCADE 7.2.0. I think, the change is correct, but I would not hurt to test it properly. Can you do it? I don't have OCCT 7.2.0 right now and I am sitting in the train and cannot install it.

Reply:
btw, do you know what the first parameter does? i just read the docs and they don't tell me.

Reply:
I don't know either ;) Thats why I want the function to be tested :D